HermaFx.DataAnnotations/NotDefaultAttribute.cs

```diff
@@ -11,17 +11,21 @@ public class NotDefaultAttribute : ValidationAttribute
 	{
 		private const string _defaultErrorMessage = "The field {0} requires a non-default value";
 
-		public NotDefaultAttribute()
-				: base(string.Format(_defaultErrorMessage))
-		{
-		}
-
 		public override bool IsValid(object value)
 		{
 			if (value == null) return true;
 			if (!value.GetType().IsValueType) return true;
 
-			return value == value.GetType().GetDefaultValue();
+			return !object.Equals(value, value.GetType().GetDefault());
+		}
+
+		public override string FormatErrorMessage(string name)
```

Comment:
This method overriding is not really needed (once you pass the desired error message format string to base class' .ctor)